mlserver/rockcraft.yaml

```diff
@@ -0,0 +1,128 @@
+# Based on https://github.com/SeldonIO/MLServer/blob/1.3.5/Dockerfile
+# Operator version is 1.17.1 version for this ROCK can be found in
+# https://github.com/SeldonIO/seldon-core/blob/v1.17.1/operator/config/manager/configmap.yaml
+name: mlserver
+summary: An image for Seldon MLServer Huggingface
+description: |
+  This image is used as part of the Charmed Kubeflow product.
+version: 1.3.5_20.04_1 # <upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:20.04
+run-user: _daemon_
+services:
+  mlserver:
+    override: replace
+    summary: "mlserver service"
+    startup: enabled
+    command: bash -c 'export PATH=/opt/conda/bin/:/opt/mlserver/.local/bin:${PATH}:/usr/bin && export PYTHONPATH=/opt/mlserver/.local/lib/python3.8/site-packages/:${PYTHONPATH} && eval $(/opt/conda/bin/conda shell.bash hook 2> /dev/null) && mlserver start ${MLSERVER_MODELS_DIR}'
+    working-dir: "/opt/mlserver"
+    environment:
+      MLSERVER_ENV_TARBALL: "/mnt/models/environment.tar.gz"
+      MLSERVER_MODELS_DIR: "/mnt/models/"
+      LD_LIBRARY_PATH: "/usr/local/nvidia/lib64:/opt/conda/lib/python3.8/site-packages/nvidia/cuda_runtime/lib:$LD_LIBRARY_PATH"
+      TRANSFORMERS_CACHE: "/opt/mlserver/.cache"
+      NUMBA_CACHE_DIR: "/opt/mlserver/.cache"
+platforms:
+  amd64:
+
+parts:
+  mlserver:
+    plugin: nil
+    source: https://github.com/SeldonIO/MLServer
+    source-type: git
+    source-tag: 1.3.5
+    build-packages:
+      - bash
```

Comment:
you already have bash, and maybe same for tar

Reply:
Will remove.

Reply:
`tar` is in upstream, keeping it.

Reply:
but i think tar is already on the base

Reply:
same for gzip normally